app/assets/javascripts/segment_history_graph.coffee

```diff
@@ -0,0 +1,41 @@
+$ ->
+  if gon.run.program isnt "livesplit"
+    return
+
+  colors_obj = {}
+  $(".timeline").children(".split").each (index) ->
+    color = $(this).css("background-color")
+    return if color is "rgba(0, 0, 0, 0)"
+    colors_obj[$(this).children()[0].innerHTML] = color
```

Comment:
it feels pretty dirty to be piggybacking colors on other html elements' colors. if the structure of the timeline changes a little bit in the wrong way, the graphs break and it would be easy not to notice anything is broken because they're on a different page from the one that the person making the change would be looking at. this also makes it impossible to have a graph on a page that doesn't also have a timeline.